### DISCUSSION TITLE
Allow access to getSessionId

### DISCUSSION THREAD

**Discussion by joelshafferunion** [firebase/flutterfire#10599] [Feature request]
There is currently no way to get the session id or number for Firebase Analytics. 

This is a feature available in Android Firebase here: https://firebase.google.com/docs/reference/android/com/google/firebase/analytics/FirebaseAnalytics#public-tasklong-getsessionid

and in iOS Firebase here:
https://firebase.google.com/docs/reference/ios/firebaseanalytics/api/reference/Classes/FIRAnalytics#/c:objc(cs)FIRAnalytics(cm)sessionIDWithCompletion:

**Comment by brunovsiqueira** [comment]
Getting this value can help app developers to do cross environment trackings (e.g. Posting events using Google's measutement protocol). Is there any plans to bring this feature, that is available in React Native too?
